core/db/entities.py

```diff
@@ -107,6 +121,27 @@ class FundingQuestion(BaseModel):
     )
 
 
+class GeospatialDim(BaseModel):
+    """Stores Geospatial information mapped to postcodes."""
+
+    __tablename__ = "geospatial_dim"
+
+    postcode_prefix = sqla.Column(sqla.String(length=4), nullable=False, unique=True)
+    itl1_region_code = sqla.Column(sqla.String(), nullable=False, unique=False)
+    data_blob = sqla.Column(JSONB, nullable=True)
```

Comment:
I don't think we ever expect this to be null, right? Should we enforce this?

Reply:
Hmmm yes this is a really good point, we should enforce it! Will update.

Reply:
Oh I'm going to remove the data_blob anyway in a separate PR as we discussed with Noel, to move the `itl1_region_name` to its own column, but yes probably good practice to enforce this in this PR even if we're going to remove it

Reply:
oh right yep 😂